rust/actyx/swarm/harness/Cargo.toml

```diff
@@ -5,16 +5,16 @@ version = "0.1.0"
 authors = ["Actyx AG"]
 
 [dependencies]
-actyx_sdk = { path = "../../../sdk", features = [
-    "client",
-    "arb",
-    "with-tokio",
-] }
-ax = { path = "../../ax" }
+ax_sdk = { path = "../../../sdk", features = ["client", "arb", "with-tokio"] }
+ax_core = { path = "../../ax-core" }
 anyhow = "1.0.52"
-async-global-executor = { version = "2.0.2", features = ["tokio"] }
+async-global-executor = { version = "2.0.2", default-features = false, features = [
+    "tokio",
+] }
+# required to avoid building rustix 0.37, which fails on macos
```

Comment:
Interesting, is there any open issue on their repo that we could add to track next time one of us looks at this file?

Reply:
Didn’t check; I’d prefer async-std to be gone instead.